.betterer.results

```diff
@@ -326,7 +326,7 @@ exports[`no enzyme tests`] = {
     "public/app/plugins/datasource/elasticsearch/configuration/DataLinks.test.tsx:2916632804": [
       [1, 17, 13, "RegExp match", "2409514259"]
     ],
-    "public/app/plugins/datasource/grafana-azure-monitor-datasource/components/InsightsConfig.test.tsx:866257119": [
```

Comment:
what does this do?

Reply:
this new tool, betterer, checks that we are not adding new unit tests based on "enzyme" this is the list of files that still use it, since I am changing the path of this file, I needed to update this list so it does not complain as a new appearance of enzyme.

Reply:
(see: https://drone.grafana.net/grafana/grafana/53211/1/12)